Ports/lcms2/package.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=lcms2
+version=2.15
+files=(
+    "https://github.com/mm2/Little-CMS/archive/refs/tags/lcms${version}.tar.gz 30d928c9b564c670b6b57c917e5c2599247b713ba6cf2dbd089a8efa70a1952f"
+)
+
+workdir="Little-CMS-lcms${version}"
+
+useconfigure=true
+
+configure() {
+  run ./"$configscript" --host=x86_64-pc-linux-gnu --target="${SERENITY_ARCH}-pc-serenity" "${configopts[@]}"
```

Comment:
This looks incorrect. We want the resulting binary to run on SerenityOS, so `--host` should still be SerenityOS. `--target` only really applies to toolchains, which can have three different machines that can be involved.

Reply:
I'm away from home so can't get deep into this, but I think that this `configure` is not "normal". I started by copying another configuration but the script, but it really insisted that I also pass a --target when using --host

Reply:
This script worked on my machine here:
```
#!/usr/bin/env -S bash ../.port_include.sh
port='lcms2'
version='2.15'
useconfigure='true'
use_fresh_config_sub='true'
use_fresh_config_guess='true'
config_sub_paths=("config.sub")
config_guess_paths=("config.guess")
files=(
    "https://github.com/mm2/Little-CMS/releases/download/lcms2.15/lcms2-${version}.tar.gz b20cbcbd0f503433be2a4e81462106fa61050a35074dc24a4e356792d971ab39"
)
depends=(
    'libtiff'
)
```

Reply:
Ok, looks like I don't know how to use configure :sweat_smile:
Go for it then!